tdp/core/deployment/test_deployment_runner.py

```diff
@@ -168,16 +176,17 @@ def test_service_log_emitted_once_with_multiple_config_and_start_on_same_compone
     minimal_collections, deployment_runner
 ):
     """deployment plan containing multiple config, start, and restart should emit only one service log"""
-    deployment_plan = DeploymentPlan.from_operations(
+    deployment_log = DeploymentLog.from_operations(
         minimal_collections,
         [
             "mock_node_config",
             "mock_node_start",
             "mock_node_config",
+            "mock_node_start",
```

Comment:
Why adding a `start` action ? It doesn't make sense to have a `start` before a `restart`